src/Microsoft.DotNet.Interactive.HttpRequestParser/Parser/HttpRequestParser.cs

```diff
@@ -697,9 +694,9 @@ private HttpHeaderValueNode ParseHeaderValue()
             return node;
         }
 
-        private HttpCommentNode? ParseComment()
+        private IEnumerable<HttpCommentNode> ParseComments()
         {
-            if (IsComment())
+            while (IsComment())
```

Comment:
We may have another bug if there is a blank line (or a line with only whitespaces) in between two comments.